packages/open-api-gateway/README.md

```diff
@@ -433,6 +438,66 @@ def handler(input: SayHelloRequest, **kwargs) -> ApiResponse[HelloResponse, ApiE
     )
 ```
 
+### Interceptors
+
+The lambda handler wrappers allow you to pass in a _chain_ of handler functions to handle the request. This allows you to implement middleware / interceptors for handling requests. Each handler function may choose whether or not to continue the handler chain by invoking `chain.next(input, event, context)`. Note that the last handler in the chain (ie the actual request handler which transforms the input to the output) should not call `chain.next`.
+
+#### Typescript
+
+In typescript, interceptors are passed as separate arguments to the generated handler wrapper, in the order in which they should be executed.
+
+```ts
+import { sayHelloHandler, ApiError } from "my-api-typescript-client";
+
+// Interceptor to wrap invocations in a try/catch, returning a 500 error for any unhandled exceptions.
+const tryCatchInterceptor = async (input, event, context, chain) => {
```

Comment:
We dont have types for this interceptor however we do for timingInteceptor. Is this intended?

Reply:
Ah was just for brevity, but probably clearer if I include the types :)